packages/qwik/src/core/util/log.ts

```diff
@@ -8,7 +8,15 @@ const STYLE = qDev
   : '';
 
 export const logError = (message?: any, ...optionalParams: any[]) => {
-  return createAndLogError(true, message, ...optionalParams);
+  const error = createAndLogError(true, message, ...optionalParams);
+
+  // make sure to throw the error in test mode so that the test fails
+  // TODO: uncomment the following to fix any unit test that MUST fail and catch it
```

Comment:
To do this correctly, you need to have async localstorage, and that's not supported in the browser environment.

I propose you revert this change?